frontend/src/components/TicketCardClose.tsx

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import {Card, CardContent, Typography, Box} from '@mui/material';
+import {styled} from '@mui/system';
+
+const Root = styled(Card)({
+  borderRadius: '16px',
+  overflow: 'hidden',
+  backgroundColor: '#FFC107',
+  width: '441.25px',
+  height: '211.25px',
+  margin: '16px auto',
+  boxShadow: '0 4px 16px rgba(0,0,0,0.2)',
+  color: '#fff',
+});
+
+const Header = styled(Box)({
+  backgroundColor: '#FFC107',
+  padding: '8px 32px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  borderTopLeftRadius: '16px',
+  borderTopRightRadius: '16px',
+  color: '#fff',
+  fontWeight: 'bold',
+  position: 'absolute',
+  zIndex: 1,
+});
+
+const HeaderText = styled(Typography)({
+  color: '#fff',
+  fontWeight: 'bold',
+  position: 'absolute',
+  top: '-7px',
+  left: '16px',
+  backgroundColor: '#FFC107',
+  padding: '4px 16px',
+  borderRadius: '12px',
+  boxShadow: '0 2px 4px rgba(0,0,0,0.2)',
+  zIndex: 20,
+});
+
+const Content = styled(CardContent)({
+  padding: '38px',
+  backgroundColor: '#FFC107',
+  color: '#fff',
+  paddingBottom: '0px',
+});
+
+const BusInfo = styled(Box)({
+  padding: '8px 16px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  borderBottomLeftRadius: '16px',
+  borderBottomRightRadius: '16px',
+  backgroundColor: '#FFC107',
+  marginTop: '8px',
+});
+
+const BusNumber = styled(Typography)({
+  fontSize: '1.5rem',
+  fontWeight: 'bold',
+  color: '#fff',
+});
+
+const TimeDate = styled(Box)({
+  textAlign: 'right',
+  color: '#fff',
+});
+
+const DashedLine = styled(Box)({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  margin: '16px 0',
+});
+
+const DashedSegment = styled(Box)({
+  width: '20px',
+  height: '0px',
+  borderTop: '2px solid #fff',
+  margin: '0 5px',
+});
+
+const Circle = styled(Box)({
+  width: '12px',
+  height: '12px',
+  borderRadius: '50%',
+  border: '2px solid #fff',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const InnerCircle = styled(Box)({
+  width: '6px',
+  height: '6px',
+  borderRadius: '50%',
+  backgroundColor: '#fff',
+});
+
+const LocationInfo = styled(Box)({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  paddingBottom: '8px',
+});
+
+const Location = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const DoneStamp = styled(Box)({
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%) rotate(-20deg)',
```

Comment:
DampStamp will not remain at the center for all screen sizes unless you set the `position: 'relative'` in root component.

Reply:
@ritankarsaha you need to change the `overflow: 'Visible'` in the root. Also, keep the borderRadius of top corner to 24px to avoid being set to zero due to visible overflow.